docs/API-Reference/bpmn-js/importXML.md

```diff
@@ -10,3 +10,62 @@ grand_parent: "API Reference"
 ---
 
 # importXML
+
+`importXML` API parses and renders a BPMN 2.0 diagram.
+
+## Usage
+
+importXML(xml: _[String](#)_, bpmnDiagram: _[ModdleElement](#)_ or _[String](#)_): Promise<_[ImportXMLResult](#importxmlresult)_, _[ImportXMLError](#importxmlerror)_>
+
+* `xml`: The BPMN 2.0 XML.
+* `bpmnDiagram`: The BPMN diagram or the id of the diagram to render _(if not provided, the first one will be rendered)_.
+
+### Code Example
+
+```javascript
+const modeler = new BpmnJS();
+
+try {
+
+  const result = await modeler.importXML(xml);
+
+  const { warnings } = result;
+
+  console.log('Import warnings: ', warnings);
+} catch (err) {
+
+  console.log('Error happened: ', err.message, err.warnings);
+}
+```
+
+### Life-Cycle Events
+
+During the import, these [events](/events) will be fired:
+
+* [import.parse.start](/events/bpmn-events/import-parse-start): About to read model from xml
```

Comment:
Let's discuss: I believe the event names are speaking for themselves, the descriptions at the end of each line in the next esection can therefore be removed [since currently they bascailly just say the same thing as the name, but with different terms]

Reply:
Yeah it makes sense. I removed the explanations 👍